### PR TITLE
Updated the documentation with a workaround to delete multiple items

### PR DESCRIPTION
**Pull request by diegomedina248**
Updated the documentation with a workaround to delete multiple items using filter, which currently is not directly supported by the SimpleNoSQL API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,11 +122,21 @@ bucket.newQuery()
 
 #### Multiple
 
+Currently, SimpleNoSQL does not support the usage of `filter` for the `delete` operation.
+There's an issue [opened](https://github.com/Jearil/SimpleNoSQL/issues/34)
```
Comment (bryant1410) [inline README.md]: Add a dot at the end :smile:

**Comment by bryant1410** [comment]
great work @diegomedina248 !

**Comment by enrigalmig** [comment]
👍

**Comment by bryant1410** [comment]
@diegomedina248 please rebase this with master :smile:

**Pull request merged**